README.md

```diff
@@ -11,15 +11,28 @@ If you are looking for documentation around the companion applications check out
 
 - Create a Firebase project at [Firebase Console](https://console.firebase.google.com)
 
-- Create two Android apps, one with `io.homeassistant.companion.android` and one with `io.homeassistant.companion.android.debug` as package name
+- Create four Android apps, with following package names 
+ - `io.homeassistant.companion.android`
+ - `io.homeassistant.companion.android.debug`
+ - `io.homeassistant.companion.android.minimal`
+ - `io.homeassistant.companion.android.minimal.debug`
```

Comment:
You should mock out the minimal packages and not need to add them in firebase.

Reply:
When I tried to build it, I got an compile error with message about these non-minimal packages missing.